compiler/luci/pass/src/CircleOptimizer.cpp

```diff
@@ -217,6 +217,19 @@ CircleOptimizer::Options *CircleOptimizer::options(void)
   return _options.get();
 }
 
+void CircleOptimizer::canonicalize(loco::Graph *g) const
+{
+  logo::Phase phase;
+
+  phase.emplace_back(std::make_unique<luci::CanonicalizePass>());
+  phase.emplace_back(std::make_unique<logo::RemoveDeadNodeWithQueryPass>());
```

Comment:
How about adding shape/dtype inference passes? If canonicalization updates dtype/shape, those passes will be needed too.